src/test/java/com/swiftype/appsearch/ClientTest.java

```diff
@@ -2,10 +2,7 @@
 
 import java.security.InvalidKeyException;
 import java.security.SignatureException;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
+import java.util.*;
```

Comment:
Usually we prefer to list out all the imports explicitly and let the IDE handle collapsing them when working with a file